<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settingsDialog</class>
 <widget class="QDialog" name="settingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>450</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>601</width>
     <height>431</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>90</y>
       <width>601</width>
       <height>341</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="paths">
      <attribute name="title">
       <string>Tab 1</string>
      </attribute>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>571</width>
         <height>131</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="title">
        <string>cl compiler environment variables</string>
       </property>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>541</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string>When using Windows, cl compiler can't be used without setting some environment variables.
 If you experience these problems, please set the following to paths:</string>
        </property>
       </widget>
       <widget class="QLabel" name="vsLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>181</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Visual Studio Installation Dir:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="vsLine">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>70</y>
          <width>331</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="sdkLine">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>100</y>
          <width>331</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="sdkLabel">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>100</y>
          <width>121</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Microsoft SDK Dir:</string>
        </property>
       </widget>
       <widget class="QPushButton" name="vsButton">
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>70</y>
          <width>31</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
       <widget class="QPushButton" name="sdkButton">
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>100</y>
          <width>31</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>571</width>
         <height>161</height>
        </rect>
       </property>
       <property name="title">
        <string>OpenCL environment variables</string>
       </property>
       <widget class="QLabel" name="clLabel">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>50</y>
          <width>81</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>CL/cl.h path:</string>
        </property>
       </widget>
       <widget class="QLabel" name="LibClLabel">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>90</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>libOpenCL path:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="clLine">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>50</y>
          <width>331</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="libClLine">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>90</y>
          <width>331</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="clButton">
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>50</y>
          <width>31</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
       <widget class="QPushButton" name="libClButton">
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>90</y>
          <width>31</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Tab 2</string>
      </attribute>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>591</width>
         <height>161</height>
        </rect>
       </property>
       <property name="title">
        <string>Running Workflow</string>
       </property>
       <widget class="QCheckBox" name="saveFlow">
        <property name="geometry">
         <rect>
          <x>4</x>
          <y>60</y>
          <width>201</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Save Running Workflow</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_16">
        <property name="geometry">
         <rect>
          <x>4</x>
          <y>30</y>
          <width>521</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>This option enables recovery in case of a system crash during execution.</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_17">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>60</y>
          <width>291</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;Warning: This will decrease performance!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Compiler Settings</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>20</y>
       <width>125</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../resources/icons.qrc">:/images/settings_logo.png</pixmap>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QTabWidget" name="tabWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>90</y>
       <width>581</width>
       <height>341</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Tab 1</string>
      </attribute>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>541</width>
         <height>201</height>
        </rect>
       </property>
       <property name="title">
        <string>General</string>
       </property>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>15</y>
          <width>61</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Tab Size</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>20</y>
          <width>51</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>161</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Turn tabs into spaces</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>60</y>
          <width>70</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QFontComboBox" name="fontComboBox">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>90</y>
          <width>187</width>
          <height>22</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Courier</family>
         </font>
        </property>
        <property name="currentFont">
         <font/>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>92</y>
          <width>46</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Font:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>130</y>
          <width>81</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Font size:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="font_size">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>130</y>
          <width>51</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>272</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Reset to defaults</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Tab 2</string>
      </attribute>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>30</y>
         <width>131</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QListWidget" name="listWidget_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>201</width>
         <height>181</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="chooseColor">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>230</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Choose color</string>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="plainTextEdit">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>100</y>
         <width>201</width>
         <height>181</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>290</y>
         <width>411</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Tip: Keywords must be separated by  space character</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>270</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Reset to default</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Language:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>181</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Groups</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>80</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Regex used</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_2">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>30</y>
         <width>321</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Highlight white spaces before new line</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Editor Settings</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>20</y>
       <width>125</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../resources/icons.qrc">:/images/settings_logo.png</pixmap>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3"/>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>151</width>
     <height>351</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>101</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(159, 159, 159);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../resources/icons.qrc">:/images/icon.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../resources/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>settingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>settingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
